lib/fathead/mdn_css/parse.pl

```diff
@@ -139,19 +139,6 @@
     $description = build_abstract( $description, $code, $initial_value );
 
     next unless $title && $link && $description;
-
```

Comment:
I had began working on this on a new [PR](https://github.com/duckduckgo/zeroclickinfo-fathead/pull/312/commits/aa5371a5015c493bba5ea1cc0b0c112d465ff4ce).

Should I abandon this feature as well over there?